src/main/java/seedu/address/model/appointment/Date.java

```diff
@@ -62,6 +65,20 @@ public boolean isAfter(Date inputDate) {
         return value.isAfter(inputDate.value);
     }
 
+    /**
+     * Checks if the appointment date is in the same week with the input {@code Date}.
+     *
+     * @return true if the appointment date is in same week with the input date, false otherwise.
+     */
+    public boolean isInSameWeek(Date inputDate) {
+        TemporalAdjuster adj = TemporalAdjusters.next(DayOfWeek.MONDAY);
```

Comment:
Would it be better if the variable name `adj` is changed into something clearer?